source/pdps-group-replication.rst

```diff
@@ -0,0 +1,233 @@
+.. _pdps-group-replication:
+
+********************************************************************************
+High Availability Solution with Group Replication. Technical Overview
+********************************************************************************
+
+.. contents::
+   :local:
+   :depth: 1
+
+Introduction
+=================
+
+Every architecture and deployment depends on customer requirements and application demands for high availability and the estimated level of usage.  For example, using a high read or a high write application, or both with  99.999% availability.
+ 
+This document gives architecture and deployment recommendations along with a technical overview for a solution that provides a high level of high availability and assumes the usage of high read / write applications (20K or more queries per second). For step-by-step deployment guidelines, refer to :ref:`deploy`.
+
+This solution assumes the use of |PS| based deployment variant of Percona Distribution for MySQL with `Group Replication <https://dev.mysql.com/doc/refman/8.0/en/group-replication.html>`_.
+
+.. _layout:
+
+Architecture layout
+=====================
+
+.. image:: _static/images/group_replication_ha.png
+
+Components
+----------------
+
+The architecture is composed of two main layers:
+
+- :ref:`Connection and distribution layer <connection>`
+- :ref:`Relational Database Management System (RDBMS) layer <rdbms>`
+
+.. _connection:
+
+Connection and distribution layer 
```

Comment:
Do we have a recommendation in our style guide about [capitalizing headlines](https://headlinecapitalization.com/)?
```suggestion
Connection and Distribution Layer 
```

Reply:
We do. Both in our style guide and in Google style guide, the general recommendation is to use sentence case in headings and titles.